lesson8_task3.py

```diff
@@ -0,0 +1,32 @@
+"""
+A simple calculator.
+
+Create a function called make_operation, which takes in a simple arithmetic operator as a first parameter
+(to keep things simple let it only be '+', '-' or '*') and an arbitrary number of arguments (only numbers)
+as the second parameter. Then return the sum or product of all the numbers in the arbitrary parameter. For example:
+
+the call make_operation('+', 7, 7, 2) should return 16
+the call make_operation('-', 5, 5, -10, -20) should return 30
+the call make_operation('*', 7, 6) should return 42
+"""
+
+
+def make_operation(operator: str, *args: int | float) -> int | float:
```

Comment:
Very good solution. Accepted it.
Couple things:
1) nice that you figured out *args correct type hint and not used tuple as type
2) you could try to rewrite this with newer match statement (although we didn't cover that, but that could be useful to you)
3) you could try to use something like functools.reduce for subtract and multiply operators to make your code more concise.